enhancements/console/guided-tours-crd.yaml

```diff
@@ -0,0 +1,163 @@
+  subresources:
+    status: {}
+  "validation":
+    "openAPIV3Schema":
+      description: GuidedTour is an extension for guiding user thought various workflows in the OpenShift web console
+        links.
+      type: object
+      required:
+      - spec
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: GuidedTourSpec is the desired guided tour configuration.
+          type: object
+          required:
+          - version
+          - displayName
+          - introduction
+          - tasks
+          properties:
+            version:
+              description: version definest the release version of the guided
+                tour. Each Guided Tour needs to be versioned, to accuratelly
+                reflect the workflow in particular release of OpenShift's web
+                console.
+              type: string
+              pattern: ^release-4.\d+$
+            displayName:
+              description: displayName is the display name of the Guided Tour
+              type: string
+            followingGuidedTour:
+              description: followingGuidedTour determines which Guided Tour should
+                be suggested to take by the user after finishing the current Guided Tour
+              type: string
+            duration:
+              description: duration informs user on aproximately how long the
+                Guided Tour should take to finish (optional)
+              type: integer
+            description:
+              description: description is the description of the Guided Tour (optional)
+              type: string
+            prerequisites:
+              description: prerequisites describes what are the prerequisites
+                for the Guided Tour (optional)
+              type: string
+            introduction:
+              description: introduction describes the purpose and usability of
+                the Guided Tour
+              type: object
+              required:
+              - about
+              properties:
+                about:
+                  description: about introduces the Guided Tour purpose
+                  type: string
+                usability:
```

Comment:
Would we want more flexibility that comes from a single markdown long description?